web/src/components/Header.tsx

```diff
@@ -218,6 +225,12 @@ export default function Header({ theme, toggleTheme }: HeaderProps) {
     toggleTheme();
   };
 
+  const handleUserInformationSave = async () => {
+    const profilePatchBody = { password, phoneNumber, address, profileImage };
```

Comment:
Please remove `password` and `phoneNumber`. Not should be included here.

Reply:
it is same before comment